pkg/util/helper/taint_test.go

```diff
@@ -357,3 +359,209 @@ func TestGetNoExecuteTaints(t *testing.T) {
 		})
 	}
 }
+
+func TestGetMinTolerationTime(t *testing.T) {
+	now := metav1.Now()
+	later := metav1.NewTime(time.Now().Add(5000))
+	type args struct {
+		noExecuteTaints []corev1.Taint
+		usedTolerations []corev1.Toleration
+	}
+	tests := []struct {
```

Comment:
I think it's necessary to add a use case to this test `exist mismatch taint`.

The current method may not take this into account in its implementation, and it assumes that all the taints are matched, which is actually the logic in the program, but as a tool method, this assumption is not reasonable, so the implementation of the method may need to be modified.